### PR TITLE
FEAT: video listing with categories filter and dropdown

### PR DESCRIPTION
**Pull request by partha8**
## files to check
1. [src/hooks/useFilteredVideos.js](https://github.com/partha8/weeb-culture/compare/videos-listing?expand=1#diff-a51c2179a47c0adb7ab4d28e781757b28c1628c8871f9301da9a862ca30fbd58)
2. [src/hooks/useGetVideos.js](https://github.com/partha8/weeb-culture/compare/videos-listing?expand=1#diff-00af39061133c46f00fe026a44a92bd1adcebf4318954fe8fa397792a8183c46)
3. [src/pages/Explore/Explore.js](https://github.com/partha8/weeb-culture/compare/videos-listing?expand=1#diff-20ac5bc126d958b1420a804601b86cb2c0b44910844f0345b6f785eabeb62e82)
4. [src/pages/Explore/components/CTABtn.js](https://github.com/partha8/weeb-culture/compare/videos-listing?expand=1#diff-b80b13fb379064db8a86f74de48d9600b362bf9d333999d83fea2ffddc80bcd8)
5. [src/pages/Explore/components/VideoCard.js](https://github.com/partha8/weeb-culture/compare/videos-listing?expand=1#diff-78d0d7d55b48c8afef31793e90591f5c0460d8a82ad5b45a254d5564b37e288e)
6. [src/reducers/videosReducer.js](https://github.com/partha8/weeb-culture/compare/videos-listing?expand=1#diff-9f9340b9067e5f2155aea27f29aa2d223cd62a977fac23a99b7c0f605ff6680a)

## preview
https://deploy-preview-2--sparkly-starlight-7c0532.netlify.app/explore

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
Whole thing looks cool to me.
Like the implementation of the Animation cum gif .
Even UI feels great.
Learnt few things such as CTA Btn implementation and also the way you bifurcated the code is pretty nice

File: src/hooks/useGetVideos.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import { useEffect } from "react";
+import { useStateContext } from "../context/StateProvider";
+
+export const useGetVideos = () => {
+  const { videosDispatch } = useStateContext();
+  useEffect(() => {
+    (async () => {
+      try {
```
Comment (anik31) [inline src/hooks/useGetVideos.js:9]: appreciation - nice implementation of hooks

**Review by anik31 — APPROVED** [review]
Everything is good, the filter functionality is working perfectly in the preview. The code is also good.

**Comment by partha8** [comment]
> Whole thing looks cool to me. Like the implementation of the Animation cum gif . Even UI feels great. Learnt few things such as CTA Btn implementation and also the way you bifurcated the code is pretty nice

Thank you @Dev-AkshatGupta for the review! Btw, there are no gifs used, all the animations on the homepage are done using CSS animations, SVG animation, and tsparticles package :satisfied:

File: src/hooks/useGetVideos.js (hunk 1 above)
Comment (partha8) [reply src/hooks/useGetVideos.js:9]: Thank you for the review @anik31

**Pull request merged**